command/format_test.go

```diff
@@ -78,6 +81,148 @@ func TestTableFormatter(t *testing.T) {
 	}
 }
 
+/*
+DESCRIPTION: Test to verify that the embedded struct SealStatusOutput
```

Comment:
Godoc specifies that funcs should be documented by making the func name the first word of the comment that precedes the declaration.  (Also, we don't normally use /**/ style comments.)

Reply:
Sounds good, will change this!